_includes/navbar.html

```diff
@@ -15,7 +15,7 @@
                 <img alt="JaVers logo" src="/img/logotype.svg"/>
             </a>
             <a class="logo-version" href="http://search.maven.org/#artifactdetails|org.javers|javers-core|{{site.javers_current_version}}|" >
-                {{site.javers_current_version}}
+                {{site.javers_current_version}} &mdash; Java 8 release
```

Comment:
Text is too long so it it's placed under the navbar on mobile. Consider adding `hidden-xs` style to `a`